Issue 88 - Update iGame.guide file

walkero-gr opened the pull request:
With this PR I updated the iGame.guide, describing all the GUI menu items, windows, tooltypes, and updated all the sections of the document to the latest changes.

Added also a changelog file. The format of this changelog file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). Tried to add as much information as I could find for the previous versions.

coldacid commented on required_files/iGame.guide:
```diff
@@ -1,325 +1,360 @@
 
-But not just whdload games. You can add and launch any type of executable, though
-it mostly makes sense if it's a game ;-)
+iGame can "discover" your games on pre-defined repositories and create a small database. You can then categorize each game according to it's genre, provide a small screenshot image to be displayed when the game is selected and quickly find the one you're looking for, using the filter gadget.
```
"it's" should be "its" (without apostrophe) since this use is not a contraction of "it is".

coldacid commented on required_files/iGame.guide:
```diff
@@ -1,325 +1,360 @@
 
-  3. Make sure the igame.iff file is in the same dir as the binary.
+@{lindent 3}
+iGame saves it's support files on it's own folder (PROGDIR:). We recommend you to install it on it's own separated directory in your hard disk.
```
These should all be "its".

coldacid commented on required_files/iGame.guide:
```diff
@@ -1,325 +1,360 @@
+This window opens from the menu "Actions > Add a Game". This is useful in cases you want to add a WDHLoad game/demo or any other game/demo which can be started from Workbench/Shell. Depending the AmigaOS version it uses compatible methods to run the executable.
 
-- Fix bugs!
+By this window you can set the "Title" of the game/demo, select the path on HD of the executable and set it's "Genre".
```
"its".

coldacid commented on required_files/iGame.guide:
```diff
@@ -1,325 +1,360 @@
-@NODE "HIST" "Version history"
+@NODE "WINSETS" "Settings Window"
+@{lindent 3}
+The iGame settings window opens from the menu item "Settings > Settings...". This configuration selections override the iGame tooltypes set on it's icon. These are saved on a text file named "igame.prefs", inside iGame folder. If this file is missing, iGame defaults to tooltypes.
```
"its".

coldacid commented on required_files/iGame.guide:
```diff
@@ -1,325 +1,360 @@
+@{b}SCREENSHOT=WIDTHxHEIGHT@{ub} sets the screenshot width and height in pixels
+@{b}FILTERUSEENTER@{ub} sets the filter field to be initiated with enter
+@{b}SAVESTATSONEXIT@{ub} saves game statistics when iGame is closed
+@{b}TITLESFROMDIRS@{ub} gets the game/demo title from it's parent directory
```
"its".

coldacid reviewed (changes requested):
I found a number of possessives with apostrophes that should be corrected.

coldacid commented:
Adding a changelog is a good idea, but it's probably something that should have been added directly into master, or through a separate PR so that this stays focused on the guide file.

walkero-gr commented:
@coldacid Thank you so much for taking the time and check this PR. I will do all the proposed changes.

I added the changelog in this PR because I removed this part from the guide. We can add more info on that in the future. I need to check all the commits for the versions I miss on this changelog, and add some detail. Not crucial though for a new release
